examples/control_character/run.py

```diff
@@ -0,0 +1,193 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# Before running this file, rename user_config.yaml.example -> user_config.yaml and modify it with appropriate paths for your system.
+
+import argparse
+import os
+import numpy as np
+import pandas as pd
+import spear
+
+
+def get_data_frame(skeletal_mesh_transforms):
```

Comment:
- This function is too abstract and it interferes with readability. Something more hard-coded would be more clear:

```
def get_data_frame(transforms):
    for transform_name, transform in transform.items():
        df = pd.DataFrame(
            columns=[
                transform_name + "_location_x",
                transform_name + "_location_y",
                transform_name + "_location_z",
                #...
                ]
            data={
                transform_name + "_location_x" : transform["location"]["x"]
                transform_name + "_location_y" : transform["location"]["y"]
                transform_name + "_location_z" : transform["location"]["z"]
                #...
                })
```